exonum/src/blockchain/builder.rs

```diff
@@ -84,15 +63,6 @@ impl BlockchainBuilder {
         self
     }
```

Comment:
@alekseysidorov @popzxc what about `.with_additional_runtime()` -> `.with_runtime()`?

Reply:
What about merging `with_runtime` and `with_external_runtime`?